platform/android/export/export.cpp

```diff
@@ -795,31 +869,125 @@ class EditorExportPlatformAndroid : public EditorExportPlatform {
+			if (hand_tracking_index > 0) {
+				if (perms->find("com.oculus.permission.HAND_TRACKING") == -1) {
+					perms->push_back("com.oculus.permission.HAND_TRACKING");
+				}
+			}
 		}
-		return perms;
 	}
 
 	Error _fix_manifest_plaintext(const Ref<EditorExportPreset> &p_preset, String &manifest_path, bool p_give_internet) {
-		//TODO: replicate the functionality of _fix_manifest by editing the plaintext AndroidManifest.xml file
+		String manifest_text = ANDROID_MANIFEST_TEXT;
+
+		String package_name = p_preset->get("package/unique_name");
+		manifest_text = manifest_text.replace("*PACKAGE_NAME*", get_package_name(package_name));
+
+		String version_name = p_preset->get("version/name");
+		int version_code = p_preset->get("version/code");
+		manifest_text = manifest_text.replace("*VERSION_NAME*", version_name);
+		manifest_text = manifest_text.replace("*VERSION_CODE*", itos(version_code));
+
+		bool min_gles3 = ProjectSettings::get_singleton()->get("rendering/quality/driver/driver_name") == "GLES3" &&
+						 !ProjectSettings::get_singleton()->get("rendering/quality/driver/fallback_to_gles2");
+		String gles_version = min_gles3 ? "0x00030000" : "0x00020000";
+		manifest_text = manifest_text.replace("*GLES_VERSION*", gles_version);
+
+		int orientation = p_preset->get("screen/orientation");
+		String orientation_name;
+		if (orientation == 0) {
+			orientation_name = "landscape";
+		}
+		if (orientation == 1) {
+			orientation_name = "portrait";
+		}
+		manifest_text = manifest_text.replace("*SCREEN_ORIENTATION*", orientation_name);
+
+		String screen_support_small = bool_to_string(p_preset->get("screen/support_small"));
+		String screen_support_normal = bool_to_string(p_preset->get("screen/support_normal"));
+		String screen_support_large = bool_to_string(p_preset->get("screen/support_large"));
+		String screen_support_xlarge = bool_to_string(p_preset->get("screen/support_xlarge"));
+		manifest_text = manifest_text.replace("*SMALL_SCREENS*", screen_support_small);
+		manifest_text = manifest_text.replace("*NORMAL_SCREENS*", screen_support_normal);
+		manifest_text = manifest_text.replace("*LARGE_SCREENS*", screen_support_large);
+		manifest_text = manifest_text.replace("*X_LARGE_SCREENS*", screen_support_xlarge);
+
+		Vector<String> perms;
+		_get_permissions(p_preset, p_give_internet, &perms);
+		String permission_string;
+		for (int i = 0; i < perms.size(); i++) {
+			permission_string = permission_string.insert(0, "\t<uses-permission android:name=\"" + perms.get(i) + "\"/>\n");
+		}
+		manifest_text = manifest_text.replace("*PERMISSIONS*", permission_string);
+
+		int xr_mode_index = p_preset->get("xr_features/xr_mode");
+		String focus_awareness = bool_to_string(p_preset->get("xr_features/focus_awareness"));
+		String plugins_names = get_plugins_names(get_enabled_plugins(p_preset));
+		if (plugins_names.empty()) {
+			plugins_names = "plugins_value";
+		}
+		String feature_string;
+
+		if (xr_mode_index == 1 /* XRMode.OVR */) {
```

Comment:
There should be code in the else case something like this:

`manifest_text = manifest_text.replace("*XR_MODE_TAG*", "");`